### ISSUE TITLE
Support for Keys F13-F24

### ISSUE THREAD

**Issue by drantis** [jinput/jinput#247]
Hello JInput Team,

I am currently working on a project that requires the use of function keys beyond the standard F1-F12, specifically F13 through F24. However, I have noticed that JInput does not currently support these keys and beyond the standard F1-F12, the current source code only mentions F13-F15

**Request:**
Would it be possible to extend the JInput library to include support for the additional function keys F13-F24?

**Comment by Valkryst** [comment]
I _think_ JInput can already support all of those keys, but you would need to update all of the plugins to include mappings for them and test on each platform. That's just from a cursory glance at the code, so please correct me if I'm wrong @Endolf.

* [Component](https://github.com/jinput/jinput/blob/d9a5a7f71f4400b09eeaec53d6c56e753e82d6bf/coreAPI/src/main/java/net/java/games/input/Component.java#L694) has F1-F15.
* [DIIdentifierMap](https://github.com/jinput/jinput/blob/d9a5a7f71f4400b09eeaec53d6c56e753e82d6bf/plugins/windows/src/main/java/net/java/games/input/DIIdentifierMap.java#L132) has F1-F15.
* [KeyboardUsage](https://github.com/jinput/jinput/blob/d9a5a7f71f4400b09eeaec53d6c56e753e82d6bf/plugins/OSX/src/main/java/net/java/games/input/KeyboardUsage.java#L145) has F1-F24.
  * Seems to pull from `Component.Identifier.Key` which only maps F1-F15, and then has some manual mappings for keys beyond F15.
* [LinuxNativeTypesMap](https://github.com/jinput/jinput/blob/d9a5a7f71f4400b09eeaec53d6c56e753e82d6bf/plugins/linux/src/main/java/net/java/games/input/LinuxNativeTypesMap.java#L113) has F1-F20.
  * Seems to pull from `Component.Identifier.Key` which only maps F1-F15.
  * Another part of the code seems to pull from `NativeDefinitions` which maps F1-F24.
* [NativeDefinitions](https://github.com/jinput/jinput/blob/d9a5a7f71f4400b09eeaec53d6c56e753e82d6bf/plugins/linux/src/main/java/net/java/games/input/NativeDefinitions.java#L207) has F1-F24
* [RawIdentifierMap](https://github.com/jinput/jinput/blob/d9a5a7f71f4400b09eeaec53d6c56e753e82d6bf/plugins/windows/src/main/java/net/java/games/input/RawIdentifierMap.java#L173) has F1-F24.

**Comment by Endolf** [comment]
> I _think_ JInput can already support all of those keys, but you would need to update all of the plugins to include mappings for them and test on each platform. That's just from a cursory glance at the code, so please correct me if I'm wrong @Endolf.

Yup, that looks about right, anyone fancy doing a PR and some testing, I don't have a keyboard with 24 F keys to test with.... :)
